src/Residue.jl

```diff
@@ -16,9 +16,14 @@ function Residue(name::String)
     return Residue(handle)
 end
 
-function Residue(topology::Topology, index::Integer)
-    handle = lib.chfl_residue_from_topology(topology.handle, UInt64(index))
-    return Residue(handle)
+function Residue(topology::Topology, index::Integer; atom=false)
```

Comment:
Yes, for me too. :smile: 
I was thinking on alternatives for that, the better one is rename for residue_for_atom.